spec/unit/parser/environment_compiler_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'puppet_spec/compiler'
 require 'puppet/parser/environment_compiler'
 
-describe "Application instantiation" do
+describe Puppet::Parser::EnvironmentCompiler do
```

Comment:
Minor nit...

With this change here, the describe labels makes less sense - for example it used to say "Application instantiation in the environment catalog ..."
Also, the use of the constant Puppet::Parser::EnvironmentCompiler instantiates it - is it used anywhere?